Automattic/jetpack
# POC: Add site_owner ID to sites API

#### Changes proposed in this Pull Request:

* For cases like Automattic/wp-calypso#56625, we need a way to check a site's owner on the front end. This adds the `site_owner` field using `wpcom_get_blog_owner()` to the sites API endpoint.
* Used in conjunction with D67910-code
* 
#### Testing instructions:

* Apply this PR and D67910-code to your sandbox
* Check the developer console https://developer.wordpress.com/docs/api/console/ - WPCOM API v1.2
* `/sites/[siteID]` should show a `site_owner` field with the correct site owner's user ID
